modules/system-checker/src/SystemCheckerView.tsx

```diff
@@ -0,0 +1,11 @@
+import { requireNativeViewManager } from 'expo-modules-core';
+import * as React from 'react';
+
+import { SystemCheckerViewProps } from './SystemChecker.types';
+
+const NativeView: React.ComponentType<SystemCheckerViewProps> =
+  requireNativeViewManager('SystemChecker');
+
+export default function SystemCheckerView(props: SystemCheckerViewProps) {
+  return <NativeView {...props} />;
```

Comment:
Does this component work?

Reply:
no, because my module doesn't implement a view